packages/salesforcedx-vscode-replay-debugger/src/adapter/apexReplayDebug.ts

```diff
@@ -122,9 +172,28 @@ export class ApexReplayDebug extends DebugSession {
     response: DebugProtocol.ContinueResponse,
     args: DebugProtocol.ContinueArguments
   ): void {
-    this.sendEvent(new ThreadEvent('exited', ApexReplayDebug.THREAD_ID));
     response.success = true;
     this.sendResponse(response);
+    while (this.logContext.hasLogLines()) {
```

Comment:
Parse log lines until we find a STATEMENT_EXECUTE whose location matches a breakpoint.